src/webui/prefjson.cpp

```diff
@@ -173,6 +175,11 @@ QByteArray prefjson::getPreferences()
     data["web_ui_username"] = pref->getWebUiUsername();
     data["web_ui_password"] = pref->getWebUiPassword();
     data["bypass_local_auth"] = !pref->isWebUiLocalAuthEnabled();
+    data["bypass_auth_subnet_whitelist_enabled"] = pref->isWebUiAuthSubnetWhitelistEnabled();
+    QStringList authSubnetWhitelistStringList;
+    for (const Utils::Net::Subnet &subnet : pref->getWebUiAuthSubnetWhitelist())
+        authSubnetWhitelistStringList << Utils::Net::subnetToString(subnet);
+    data["bypass_auth_subnet_whitelist"] = authSubnetWhitelistStringList.join("\n");
```

Comment:
Unfortunately I noticed it only now... Why have you chosen this out-of-context way to encode this list? We should send it as (json) list, isn't it? I'm sure this should be fixed soon.